### ISSUE TITLE
V8: Simplify font setup

### ISSUE THREAD

**Issue by BatJan** [umbraco/Umbraco-CMS#4474]
I discovered that the setup of the fonts.less file can be optimized. There really is no need to worry about serving .ttf and doing IE fixes etc. for instance. .woff is supported for IE10 and .woff2 is supported for every major browser (Yes, that includes Edge ;-)).

I'm also going to tweak the setup a bit so a local lookup is done on the users machine for the font and experiment with the font-display setting. This will reduce code and files and should also result in a nice little performance boost as well.

How can you help?
-------------------------------

PR coming up!

**Comment by nielslyngsoe** [comment]
Hi @BatJan 

Let me know how this is coming along, looking forward to your PR!

**Comment by BatJan** [comment]
Hey @nielslyngsoe - The PR is ready for being reviewed. It's not a biggie - I have deleted all of the legacy font types and used .woff and .woff2 only since the browsers that Umbraco support have support for these formats. For the OpenSans I have downloaded new version of the .woff and .woff2 files since the .woff2 file did not exist beforehand.

But I'm not even sure it makes sense to have OpenSans in the source code anymore since everything seems to be using Lato - They only place I was able to trace the usage of OpenSans was in the "Preview" function... But perhaps it should be using Lato too? Which will make it possible to delete OpenSans entirely? Or am I overlooking something? 😃

I have also set font-display: swap on each of the weights so we should avoid any potential flash of invisible text.

Let me know if you have any change requests etc. - The PR is available at #4475
